### PR TITLE
 Add test.py to icmp6 integration test. 

### PR DESCRIPTION
**Pull request by niks3089**
This change is [Reviewable](https://reviewable.io/reviews/hioa-cs/includeos/1767)

### CODE REVIEW

**Comment by mnordsletten** [comment]
The `Intrusive` key in the `vm.json` indicates that the test performs some sort of package/environment installation. I don't think this is necessary here.

Also in my pr: https://github.com/hioa-cs/IncludeOS/pull/1768 I've tried to resolve a few ip address conflicts that might arise when running tests in parallel. I know that `10.0.0.45` is already in use by the normal icmp test. I know that at the moment `10.0.0.51` is not in use by any other test, so you could change to use that instead.

**Comment by niks3089** [comment]
Thanks @mnordsletten, I have made the changes. Also, I was looking at the network tests console ouptut and I see that the tests marked as intrusive are skipped. So they are never run on jenkins?

**Comment by mnordsletten** [comment]
Great :) 

As it is right now they are run as a separate job every night on a newly spawned machine. https://jenkins.includeos.org/job/pr_intrusive_tests

**Comment by mnordsletten** [comment]
I might have made a mistake regarding `10.0.0.51` It seems like the vlan tests are actually using that address. Sorry about that. I know for sure that `10.0.0.52` is available though.

File: test/net/integration/icmp6/test.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "Type: ECHO REPLY (129)" in output_data and \
+    "Code: DEFAULT (0)" in output_data and \
+    "Checksum: " in output_data and \
+    "Data: INCLUDEOS12345ABCDEFGHIJKLMNOPQRSTUVWXYZ12345678" in output_data:
+    num_successes += 1
+    print color.INFO("<Test.py>"), "Ping test succeeded"
+  else:
+    print color.FAIL("<Test.py>"), "Ping test FAILED"
+
+vm.on_output("Service IPv4 address: 10.0.0.45, IPv6 address: fe80:0:0:0:e823:fcff:fef4:85bd", start_icmp_test);
```
Comment (mnordsletten) [inline test/net/integration/icmp6/test.py]: Make sure you change this line here to match the `10.0.0.52` address as well.

**Comment by mnordsletten** [comment]
I did some checking and ipv6 is indeed working. I got your test to work, but a few modifications are needed:

1. There aren't 11 lines to read output from. We got it working with range(0, 9). This might need to be more dynamic, if the number of output lines changes in the future?
2. Needs to call vm.Exit(0, "Test completed without errors"). The vm does not shut down automatically. This call can be done after print "ping test succeeded"

I think a little rewrite of the `start_icmp_test` logic might be necessary :)

**Comment by niks3089** [comment]
I need to add more tests. I've simply made the required fixes to get the tests to pass. I will try to make it dynamic when I add more tests

**Pull request merged**